CHANGELOG.rst

```diff
@@ -5,6 +5,12 @@ Change Log
    This file loosely adheres to the structure of https://keepachangelog.com/,
    but in reStructuredText instead of Markdown.
 
+2023-08-07
```

Comment:
```suggestion
2023-08-11
```

Reply:
We need to get scriv into this...  I'll fix it.